libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp

```diff
@@ -145,6 +145,13 @@ const AP_Param::GroupInfo AP_AdvancedFailsafe::var_info[] = {
     // @Units: s
     AP_GROUPINFO("RC_FAIL_TIME",   19, AP_AdvancedFailsafe, _rc_fail_time_seconds,    0),
 
+    // @Param: TERM_DLY
+    // @DisplayName: Termination Delay
+    // @Description: This is the time in seconds the aircraft will stay at a failsafe waypoint before aero termination.  Set to 0 to disable.
+    // @User: Advanced
+    // @Units: seconds 
+    AP_GROUPINFO("TERM_DLY",   20, AP_AdvancedFailsafe, _terminal_delay,   0),
```

Comment:
Parameter is declared `_terminate_delay` so you need to change `_terminal_delay` to `_terminate_delay`  (or better naming).